R/pkg/R/functions.R

```diff
@@ -3404,19 +3404,24 @@ setMethod("collect_set",
 #' Equivalent to \code{split} SQL function.
 #'
 #' @rdname column_string_functions
+#' @param limit determines the size of the returned array. If `limit` is positive,
+#'        size of the array will be at most `limit`. If `limit` is negative, the
```

Comment:
you can't use backtick in R doc

Reply:
Let's also match the doc. You can refer https://github.com/apache/spark/blob/4e84df0a80c1e610068884f937b73478be7e1c1c/R/pkg/R/functions.R#L186-L191

for list

Reply:
will do